<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Imagen</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Nivel de Grises : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtNivelGrisesImg"/>
            </item>
            <item>
             <widget class="QPushButton" name="btn_CargarConsultas">
              <property name="text">
               <string>Consultas</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_CargarDatos">
              <property name="text">
               <string>Datos</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QListWidget" name="lstQuery"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QComboBox" name="cmbType">
                <item>
                 <property name="text">
                  <string>Range</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Nearest</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txt_param"/>
              </item>
              <item>
               <widget class="QPushButton" name="btnCalcularResultados">
                <property name="text">
                 <string>Calcular</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="lstResult"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Matriz numerica</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="6" column="1">
           <layout class="QGridLayout" name="gridLayout_5">
            <item row="0" column="0">
             <widget class="QListWidget" name="lstValores"/>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Escala de Gris</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txt_EscalaGrises">
              <property name="text">
               <string>4</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Ingrese valores separados por un espacio</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <widget class="QPlainTextEdit" name="txtValoresMatriz">
              <property name="plainText">
               <string>63 62 70 110 50 40 120 110 12 135 150 175 180 152 200 220</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="4" column="1">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Angulo</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="cmbAngulo">
              <item>
               <property name="text">
                <string>0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>45</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>90</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>135</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Promedio</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Ingrese tama√±o de la matriz cuadrada</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="txtSize">
              <property name="text">
               <string>4</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="btnMatrizNumerica">
            <property name="text">
             <string>Calcular</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
